wpeverest/user-registration
# UR-73 Fix - Logout url not working in gutenberg login form

### All Submissions:






### Changes proposed in this Pull Request:



Previously, User Registration widgets did not show in Gutenberg while editing existing pages or adding a new page and When the login page or my Account page is added using Gutenberg block then the logout URL is not working. This PR will fix these issues.
### How to test the changes in this Pull Request:

1. First navigate to pages from the admin dashboard.
2. Then select existing pages or add a new page.
3. then click on add block i.e (+) sign and search for user registration and select it.
4. then on the right side you can choose form setting as Login form then select shortcode as login shortcode or My account shortcode.
5. after that you can enter the logout URL and verify whether it is working fine or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Logout URL not working in Gutenberg login form
